[docs] Deckhouse Registry mirroring docs (#6339)
deckhouse/deckhouse: docs/documentation/pages/DECKHOUSE-FAQ.md

1. Push the images using the `dhctl mirror` command to an air-gapped registry.

   Example of pushing images from the `/tmp/d8-images/d8.tar` tarball:

   ```shell
   dhctl mirror --images-bundle-path /tmp/d8-images/d8.tar --registry="your.private.registry.com:5000" --registry-login="<USERNAME>" --registry-password="<PASSWORD>"
   ```

   > Note that images will be pushed to the `/deckhouse/ee` repository in your registry. Before running the command, make sure the path `/deckhouse/ee` exists, and the account you are using has write permissions.

1. After pushing images to an isolated private registry, you are ready to install Deckhouse from it. Use the [Getting started](/gs/bm-private/step2.html) guide.

   When starting the installer, use its image from the registry, into which Deckhouse images were previously pushed, and not from the public registry. For example, use an address like `your.private.registry.com:5000/deckhouse/ee/install:stable` instead of `registry.deckhouse.io/deckhouse/ee/install:stable`.

>    During installation add the address of your registry and authorization data to the `InitConfiguration` resource (parameters [imagesRepo](/documentation/v1/installing/configuration.html#initconfiguration-deckhouse-imagesrepo), [registryDockerCfg](/documentation/v1/installing/configuration.html#initconfiguration-deckhouse-registrydockercfg) or [step 3](/gs/bm-private/step3.html) of the Getting started guide).

Reviewer: ```suggestion
   During installation, add your registry address and authorization data to the `InitConfiguration` resource (the [imagesRepo](/documentation/v1/installing/configuration.html#initconfiguration-deckhouse-imagesrepo) and [registryDockerCfg](/documentation/v1/installing/configuration.html#initconfiguration-deckhouse-registrydockercfg) parameters; you might refer to [step 3](/gs/bm-private/step3.html) of the Getting started guide as well).
```

   ```

   Note that the directory on the host will be mounted in the installer container. This directory will contain the pulled Deckhouse tarball.

1. Pull Deckhouse images using the `dhctl mirror` command.

   `dhctl mirror` pulls only the latest available patch versions of a minor Deckhouse release. For example, for Deckhouse 1.52, only one version, `1.52.10`, will be pulled, as it is sufficient to update Deckhouse from version 1.51.

   The command below will pull Deckhouse tarballs for versions that are on the release channels (check [flow.deckhouse.io](https://flow.deckhouse.io) for the current status of the release channels):

   ```shell
   dhctl mirror --license="<DECKHOUSE_LICENSE_KEY>" --images-bundle-path /tmp/d8-images/d8.tar
   ```

   To pull all Deckhouse images starting from a particular version, specify it in the `--min-version` parameter in the `X.Y` format.

   For example, here is how you can pull all Deckhouse version images starting from version 1.45:

   ```shell
   dhctl mirror --license="<DECKHOUSE_LICENSE_KEY>" --images-bundle-path /tmp/d8-images/d8.tar --min-version=1.45
   ```

>    > Note that `--min-version` parameter will be ignored if you specify version above current rock-solid channel.

Reviewer 2: ```suggestion

   > Note that `--min-version` parameter will be ignored if you specify version above current rock-solid channel.
```

   ```

   Note that the directory on the host will be mounted in the installer container. This directory will contain the pulled Deckhouse tarball.

1. Pull Deckhouse images using the `dhctl mirror` command.

   `dhctl mirror` pulls only the latest available patch versions of a minor Deckhouse release. For example, for Deckhouse 1.52, only one version, `1.52.10`, will be pulled, as it is sufficient to update Deckhouse from version 1.51.

   The command below will pull Deckhouse tarballs for versions that are on the release channels (check [flow.deckhouse.io](https://flow.deckhouse.io) for the current status of the release channels):

   ```shell
   dhctl mirror --license="<DECKHOUSE_LICENSE_KEY>" --images-bundle-path /tmp/d8-images/d8.tar
   ```

   To pull all Deckhouse images starting from a particular version, specify it in the `--min-version` parameter in the `X.Y` format.

   For example, here is how you can pull all Deckhouse version images starting from version 1.45:

   ```shell
   dhctl mirror --license="<DECKHOUSE_LICENSE_KEY>" --images-bundle-path /tmp/d8-images/d8.tar --min-version=1.45
   ```

>    

Reviewer 2: ```suggestion

```

### How to bootstrap a cluster and run Deckhouse without the usage of release channels?

> Use of this method only valid if you don't have release channel images in your air-gapped registry.

Reviewer: ```suggestion
This method should only be used if there are no release channel images in your air-gapped registry.
```
